bisq-mobile.adoc

```diff
@@ -0,0 +1,84 @@
+= Bisq Mobile
+:imagesdir: images
+:!figure-caption:
+
+There are a couple of efforts underway to enable mobile functionality for Bisq. One of those efforts is Bisq mobile notifications, and it's ready to use today.
+
+== Mobile notifications
+
+The Bisq mobile notifications apps let you get notifications from Bisq desktop on your iPhone or Android device. This doc explains how to set it up and how it works.
+
+=== 1. Get the app
+
+* iOS: _link coming soon—waiting for Apple approval_
+* Android: _link coming soon—waiting for Google approval_
+
+=== 2. Pair it with Bisq desktop
+
+In your desktop Bisq app, go to the `Account` screen and click the `Notifications` button. There, click on `Scan QR code`. 
+
+In the mobile app, click `Pair` and scan the QR code with your computer's webcam. 
+
+[NOTE]
+If you don't have a webcam, click the `I don't have a webcam` button in desktop Bisq. Then send the pairing string on your mobile device to your computer and paste it in the text field. 
+
+You'll see a success message once pairing complete, and can then proceed to the notifications screen.
+
+=== 3. Configure notifications
+
+There are several types of notifications you can get: trade activities, offers you are interested in, price alerts, and dispute alerts. You can also enable and disable notification sounds.
```

Comment:
There are only 3 groups:
- Trade messages (my offers, trade state changes, disputes)
- Offer alerts (offers i am interested in)
- Price alerts

In the follow up text that would need to be adjusted as well.